database/auth.go

```diff
@@ -2,13 +2,18 @@ package database
 
 import (
 	"fmt"
+	"time"
 
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting/exported"
+	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+	dbtypes "github.com/forbole/bdjuno/v2/database/types"
 	dbutils "github.com/forbole/bdjuno/v2/database/utils"
+	"github.com/lib/pq"
 
 	"github.com/forbole/bdjuno/v2/types"
 )
 
-// SaveAccounts saves the given accounts inside the database
+// SaveAccounts saves the given account addresses inside the database
```

Comment:
I think it's fine to leave the description as it was before. We might never know in the future if we want to add other things to each account honestly. 
```suggestion
// SaveAccounts saves the given accounts inside the database
```

Reply:
fixed